src/ios-deploy/ios-deploy.m

```diff
@@ -1823,7 +1834,7 @@ int main(int argc, char *argv[]) {
     };
     int ch;
 
-    while ((ch = getopt_long(argc, argv, "VmcdvunNrILeD:R:i:b:a:s:t:g:x:p:1:2:o:l::w::9::B::W", longopts, NULL)) != -1)
+     while ((ch = getopt_long(argc, argv, "VSmcdvunrILeD:R:i:b:a:t:g:x:p:1:2:o:l::w::9::B::W", longopts, NULL)) != -1)
```

Comment:
You broke the -s option when adding your -S option

Reply:
fixed